LEAF_Nexus/js/dialogController.js

```diff
@@ -18,10 +18,15 @@ function dialogController(containerID, contentID, indicatorID, btnSaveID, btnCan
 	this.validatorOks = new Object();
 	this.invalid = 0;
 
+	//calculate min width of dialog based on min width of content div
+	var minWidth = parseInt($('#' + this.contentID).css('min-width'));
```

Comment:
Did routing require changing the style of a dialog?

Reply:
This has dev merged into it. So there are some changes not related to routing.